main.py

```diff
@@ -1,7 +1,32 @@
 import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genre = [
+        {"name": "Western"},
+        {"name": "Action"},
+        {"name": "Dramma"},
+    ]
+    Genre.objects.bulk_create([Genre(**genre) for genre in genre])
+
+    actors = [
+        Actor(first_name="George", last_name="Klooney"),
+        Actor(first_name="Keanu", last_name="Reaves"),
+        Actor(first_name="Scarlett", last_name="Keegan"),
+        Actor(first_name="Will", last_name="Smith"),
+        Actor(first_name="Jaden", last_name="Smith"),
+        Actor(first_name="Scarlett", last_name="Johansson"),
+    ]
+    Actor.objects.bulk_create(actors)
```

Comment:
That is not good practice to create dataset with instances of some model . It is better to create just a list or tuples if you need to store more data in . And then create db data using `create() `
Bulk create also good too , but on really big sets of data ( 100 + rows for example )